tox.ini

```diff
@@ -151,19 +151,30 @@ commands_pre =
   ext: pip install {toxinidir}/opentelemetry-api
   grpc: pip install {toxinidir}/ext/opentelemetry-ext-grpc
   grpc: pip install {toxinidir}/opentelemetry-sdk
+  grpc: pip install {toxinidir}/tests/util
```

Comment:
maybe overkill, but thoughts on just pip installing these for pretty much every package globally? I feel like these explicit callouts are going to boilerplate that only serves to confuse someone who wants to contribute an extension.

Reply:
I totally agree, I also got confused by it.

I updated this to install api, sdk and utils for all the test cases, it looks much better now.